src/stores/gameList.js

```diff
@@ -8,6 +8,7 @@ class GameSummary {
     this.id = obj.id ? obj.id : null;
     this.name = obj.name ? obj.name : null;
     this.numPlayers = Object.prototype.hasOwnProperty.call(obj, 'players') ? obj.players.length : 0;
+    this.players = Object.prototype.hasOwnProperty.call(obj, 'players') ? obj.players : [];
```

Comment:
Let's also remove `numPlayers` above since its just computed from `players.length`

Reply:
We will additionally need to update the places `numPlayers` are referenced and remove them accordingly